src/ui/components/results/verticalresultscomponent.js

```diff
@@ -290,8 +290,9 @@ export default class VerticalResultsComponent extends Component {
 
     const navigationData = this.core.storage.get(StorageKeys.NAVIGATION);
     const dataTabOrder = navigationData ? navigationData.tabOrder : [];
-    const tabOrder = getTabOrder(this._verticalsConfig, dataTabOrder);
-    const params = new SearchParams(window.location.search.substring(1));
+    const tabOrder = getTabOrder(
+      this._verticalsConfig, dataTabOrder, this.core.storage.getUrlWithCurrentState());
+    const params = new SearchParams(this.core.storage.getUrlWithCurrentState());
```

Comment:
This is interesting. Whereas before we would take whatever is in the window.location, now we are explicitly using things that are in persistentStorage only. This is a change I would feel like may take people by surprise, if they were adding their own query parameters after page load without knowing they needed to add it to persistent storage as well. What do you think?

Reply:
talked offline, added getCurrentStateUrlMerged() to storage. Tested by entering `window.history.replaceState(null, '', '?a=b')` into console and then loading up noresults.